src/rust/engine/src/tasks.rs

```diff
@@ -8,9 +8,14 @@ use selectors::{Selector, Select, SelectDependencies, SelectLiteral, SelectProje
  * Registry of tasks able to produce each type, along with a few fundamental python
  * types that the engine must be aware of.
  */
+// TODO remove Clone https://github.com/pantsbuild/pants/issues/4236
+#[derive(Clone)]
```

Comment:
With the merge from master, I was forced to handle this. So it's gone. Closing the issue.